### ISSUE TITLE
Split section out from the ContentContainer component

### ISSUE THREAD

**Issue by jakewheeler** [CDCgov/phdi#2984]
The `ContentContainer` component we use to help us quickly layout pages consistently will render both a `section` and a `div`. While this made sense initially, usage of the `ContentContainer` has changed a bit over time and it'd now make sense to remove the `section` from this component and instead allow the developer to define a section if they wish to do so.

Here is an example of today's usage:
```tsx
<ContentContainer> // renders both a section and div
    <p>Text here</p> 
</ContentContainer>
```

and here is how we'd like to use it after this work is completed:
```tsx
<ContentContainer> // only renders a div
    <section id="results">
        <p>Text here</p> 
    </section>
</ContentContainer>
```

**Comment by nickbristow** [comment]
So the div contains the section not the other way round?
```
<section id="results">
 <ContentContainer>
  <p>Text here</p>
 </ContentContainer>
</section>
```

**Comment by jakewheeler** [comment]
Correct! I'm thinking the `div` can be used to handle the page layout and then, if you want to add a `section`, you can add your styles to it directly if you need any additional styling within the `section`.
